proto/trogonai/grpc_nats_micro/v1/echo.proto

```
@@ -0,0 +1,45 @@
+service Echo {
+  option (trogon.nats.micro.v1alpha1.service) = {
+    version: "1"
+  };
+
+  // Say returns the request message unchanged.
+  rpc Say(EchoRequest) returns (EchoReply);
+
+  // Fail always faults, so tests can exercise the google.rpc.Status error
+  // channel. The requested code is echoed back as the fault's status code.
+  rpc Fail(FailRequest) returns (EchoReply) {
+    option (trogon.nats.micro.v1alpha1.method) = {
```

Buf (1.72.0)

[error] cannot find `trogon.nats.micro.v1alpha1.method` in this scope

(COMPILE)